data/content/aws-stack.yml

```
@@ -474,7 +474,7 @@ Parameters:
 
   EnableInstanceStorage:
     Type: String
-    Description: Mount available NVMe Instance Storage at /mnt/ephemeral
+    Description: Mount available NVMe Instance Storage at /mnt/ephemeral, and use it to store docker images and containers, and the build working directory. You must ensure that the instance types have instance storage available for this to have any effect. See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-store-volumes.html
```

Just simplifying the sentence a little. I wanted to make the link nicer, but see that it's limited on this page.

```suggestion
    Description: Mount available NVMe Instance Storage at /mnt/ephemeral. Use it to store Docker images and containers, and the build's working directory. You must ensure that the instance types have instance storage available for this to have any effect. See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-store-volumes.html
```